suites/Operations/ProposalOperations/ProposalUpdate.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+
+import lemoncheesecake.api as lcc
+from lemoncheesecake.matching import check_that, equal_to
+from project import INIT0_PK, INIT1_PK
+
+from common.base_test import BaseTest
+
+SUITE = {
+    "description": "Operation 'proposal_update'"
+}
+
+
+@lcc.prop("main", "type")
+@lcc.tags("operations", "asset_transfer_operations", "proposal_update")
+@lcc.suite("Check work of method 'proposal_update'", rank=1)
+class ProposalUpdate(BaseTest):
+
+    def __init__(self):
+        super().__init__()
+        self.__database_api_identifier = None
+        self.__registration_api_identifier = None
+        self.echo_acc0 = None
+        self.echo_acc1 = None
+
+    def get_expiration_time(self, seconds):
+        pattern = "%Y-%m-%dT%H:%M:%S"
+        now = self.get_datetime(global_datetime=True)
+        expiration = datetime.strptime(now, pattern) + timedelta(seconds=seconds)
+        return expiration.strftime(pattern)
+
+    def setup_suite(self):
+        super().setup_suite()
+        self._connect_to_echopy_lib()
+        lcc.set_step("Setup for {}".format(self.__class__.__name__))
+        self.__database_api_identifier = self.get_identifier("database")
+        self.__registration_api_identifier = self.get_identifier("registration")
+        lcc.log_info(
+            "API identifiers are: database='{}', registration='{}'".format(self.__database_api_identifier,
+                                                                           self.__registration_api_identifier))
+        self.echo_acc0 = self.get_account_id(self.accounts[0], self.__database_api_identifier,
```

Comment:
remove default accounts and init committee_accounts (like in proposal_delete suite)